simp/simp-core
# (SIMP-6619) Update r10K & fix install_from_tar test

- Updated to R10k 3.1.1
- Updated component versions
  - Latest simp-utils
  - pupmod-simp-pupmod 7.10.1
  - pupmod-simp-simp_options 1.2.4
  - Latest pupmod-simp-ssh
- Reworked install_from_tar test to work with SIMP 6.4.0
  - Install the puppetserver RPM on the Puppet master host,
    because it is no longer pulled in as a dependency of any
    of the SIMP RPMs.
  - Changed name of simp-adapter-foss to simp-adapter, as
    the simp-adapter-foss and simp-adapter-pe have now been
    merged into simp-adapter.
  - Updated template for simp_conf.yaml for new questionnaire
  - Switched SIMP server IP to be on the private network in
    lieu of the NAT'd DHCP network, and no longer configure
    the network with `simp config`.
  - Ensure the hostname for each host is set to the FQDN.
  - Created a local site module, as the site module (skeleton)
    is no longer delivered with SIMP.
  - Removed management of sshd configuration for
    PasswordAuthentication in the site module, as this configuration
    parameter is now managed by the ssh module.
  - Adjusted all the LDIF templates to set shadowLastChange to an
    appropriate values (current time), as the accounts were being
    added with values that forced password reset.
  - Switched the environment from simp to production.
  - Adjusted messages searched for in the rsyslog tests.  Some
    application messages have changed.

SIMP-6619 #close
SIMP-6672 #close
